clamav/0.105/debian/Dockerfile

```diff
@@ -34,16 +37,21 @@ RUN apt update && apt install -y \
         libssl-dev \
         libxml2-dev \
         zlib1g-dev \
+        curl \
     && \
     rm -rf /var/cache/apt/archives \
     && \
     # Using rustup to install Rust rather than rust:1.62.1-bullseye, because there is no rust:1.62.1-bullseye image for ppc64le at this time.
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y \
     && \
+    . $CARGO_HOME/env \
+    && \
+    rustup update \
+    && \
     mkdir -p "./build" && cd "./build" \
     && \
     cmake .. \
-          -DCARGO_HOME="/src/build" \
+          -DCARGO_HOME=$CARGO_HOME \
```

Comment:
This is also missing from the commit for the unstable Dockerfile.

Reply:
fixed